src/nvim/pos.h

```diff
@@ -11,9 +14,9 @@ typedef int colnr_T;
 #define PRIdCOLNR "d"
 
 /// Maximal (invalid) line number
-enum { MAXLNUM = 0x7fffffff };
+#define MAXLNUM LONG_MAX
 /// Maximal column number, 31 bits
-enum { MAXCOL = 0x7fffffff };
+#define MAXCOL INT_MAX
```

Comment:
what's the reason for this? ZyX made it an enum intentionally.

Reply:
Consistency with MAXLNUM. `enum` is restricted to `int` values.